Source/WebCore/platform/audio/cocoa/CARingBuffer.cpp

```diff
@@ -146,7 +147,10 @@ inline void ZeroABL(AudioBufferList* list, size_t destOffset, size_t nbytes)
     while (--nBuffers >= 0) {
         if (destOffset > dest->mDataByteSize)
             continue;
-        memset(static_cast<Byte*>(dest->mData) + destOffset, 0, std::min<size_t>(nbytes, dest->mDataByteSize - destOffset));
+        auto mDataSpan = mDataMutableByteSpan(*dest).subspan(destOffset);
```

Comment:
I don't think we should have "m" in the name of this local variable. I think the "m" is for data member.